CMakeLists.txt

```diff
@@ -226,16 +226,16 @@ add_glob_target(
   FLAGS -ll ${PROJECT_SOURCE_DIR}/test/lua_runner.lua ${CMAKE_BINARY_DIR}/usr luacheck -q
   GLOB_DIRS runtime/ scripts/ src/ test/
   GLOB_PAT *.lua
-  TOUCH_STRATEGY SINGLE)
+  TOUCH_STRATEGY PER_DIR)
```

Comment:
Trying this

Reply:
Lol, I was fonna suggest we remove `PER_DIR` since we don't use it, and trying to keep `--target help` clean feels like a pipe dream at this point in time.

Reply:
Up to you. But generating thousands of touch-files (and targets) seems excessive so I thought it's worth seeing if we can be less noisy, if the computation is cheap.